.github/workflows/operator_ci.yaml

```diff
@@ -0,0 +1,183 @@
+name: Operator Build
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  Prepare_env:
+   name: "prepare env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: Setup dependencies
+        id: setup_dependencies
+        run: |
+          build/ci/github_actions/setup_ci_dependencies.sh
+        env:
+          build_number: "${{ github.run_number }}"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-${{ hashFiles('dev-requirements.txt') }}
+      - name: Save dependencies files
+        uses: actions/upload-artifact@v2
+        with:
+          name: ci-dependencies
+          path: |
+            /home/runner/.bash_profile
+          retention-days: 1
+   outputs:
+     docker_image_branch_tag: "${{ steps.setup_dependencies.outputs.docker_image_branch_tag }}"
+     operator_image_tag_for_test: "${{ steps.setup_dependencies.outputs.operator_image_tag_for_test }}"
+
+  Unit_testing:
+    runs-on: ubuntu-latest
+    name: "Unit testing"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run unit tests
+        run: |
+          mkdir -p build/_output/reports && chmod 777 build/_output/reports
+          ./build/ci/run_unittests.sh
+
+  Run_OLM_validation:
+    runs-on: ubuntu-latest
+    name: "Run OLM validation"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run OLM validation
+        run: |
+          ./build/ci/run_olm_validation.sh
+          
+  CSI-Build-and-push-images:
```

Comment:
so `Run_OLM_validation` is with underscores, but `CSI-Build-and-push-images` is with hyphens.
I know you love all conventions, but you have to give one up!

Reply:
is it just me or is starting with uppercase letter looks weird? e.g. `CSI_Build_and_push_image`
@ArbelNathan @zingero what do you think?

Reply:
I think it looks weird for you because we still have a mix of conventions.
usually, when we use underscores, we don't use uppercase letters at all.
I suggest lowering the case in these cases

Reply:
agree with @zingero

Reply:
what about `name: "CSI Build and push images"`? should we really lower CSI to csi?

Reply:
@ArbelNathan @zingero

Reply:
why we need `CSI`?

Reply:
if I understand correctly, I don't think `CSI` is correct here, since it's not the driver images

Reply:
and if we stop pushing nodeagent, it's not even `images`, it's `image`

Reply:
if indeed `CSI` is not correct, `Build and push image` sounds good

Reply:
`build_and_push_image` sounds even better

Reply:
@zingero so do you prefer all of them to start with a lowercase letter?

Reply:
If we use underscore_case, yes.

Reply:
then @matancarmeli7 let's use the agreed convention

Reply:
done

Reply:
are you sure? I [still see `CSI`](https://github.com/IBM/ibm-block-csi-operator/pull/189#discussion_r689943366)

Reply:
oh I didn't read good, I changed only this https://github.com/IBM/ibm-block-csi-operator/blob/914818a2e068fd63a8fbfbbb01bbd1f9bd4e613a/.github/workflows/operator_ci.yaml#L65 line

Reply:
the job "key" and name should be aligned regardless

Reply:
done